src/odemis/driver/tmcm.py

```diff
@@ -1559,6 +1606,8 @@ def moveAbs(self, pos):
         self._checkMoveAbs(pos)
 
         for a, p in pos.items():
+            if a in self._name_to_do_axis:
```

Comment:
The do-axes are not in referenced, so every time we move them a warning appears, which is unnecessary.

Reply:
Indeed, sorry about this. I had in mind the opposite default behaviour for the line just after... I think you can change to the following:
`if not self.referenced.value.get(a, True) and p != self.position.value[a]:`
For the  the analog axes which are not added to .referenced, this will also remove the warning. The main goal of this warning is to detect axes which should be referenced, but for some reason, someone tries to move it to an "absolute" position before it's referenced.

Reply:
Please drop this code, and update the default value for `.referenced`.